libs/community/tests/integration_tests/storage/test_sql.py

```diff
@@ -0,0 +1,186 @@
+"""Implement integration tests for Redis storage."""
```

Comment:
I'd recommend removing integration tests since they duplicate the content of unit tests.

unit tests will run on CI right now under the extended tests run.

At the moment, integration tests are not run as part of CI (since they typically require access to external resources that are not set up)